modules/persistent-storage-csi-migration-sc.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
```

Comment:
I don't actually see this in the assembly.

Reply:
Yup. I ended up not using it. I've deleted it now.